### DISCUSSION TITLE
How to get current product id in APP extension iframe when using mount point PRODUCT_DETAILS_MORE_ACTIONS?

### DISCUSSION THREAD

**Discussion by manojLondhe** [saleor/saleor#9724] [Q&A]
Hi,

I am trying to extend dashboard using docs - [Extending Dashboard with Apps](https://docs.saleor.io/docs/3.x/developer/extending/apps/extending-dashboard-with-apps) .

I am using sample app from this repo https://github.com/mirumee/saleor-app-framework-python/tree/main/samples

I want to use the mount point `PRODUCT_DETAILS_MORE_ACTIONS` (Mount extension on product's detail page under a more action button) to show few more details that we are storing on other server which are related to product.

Q1: Since app's extensions are loaded as iframe how will the data exchange between Saleor Dashboard and App will take place? For example - I want to get the current product id in the iframe when using  `PRODUCT_DETAILS_MORE_ACTIONS` this as mount point?

Q2: If that is not possible what are 1 or 2 example use cases where PRODUCT_DETAILS_MORE_ACTIONS will be used?

Thanks.

**Comment by manojLondhe** [comment]
anyone knows about this?

**Comment by korycins** [comment]
Hey, 
We don't support passing ID to iframe yet. It will be handled by [app-bridge](https://github.com/saleor/app-bridge) in future releases.
